src/styles/components/_cookieBanner.scss

```diff
@@ -0,0 +1,54 @@
+.util-cookie-banner-container {
+  flex-direction: column;
```

Comment:
Not sure why column is here? default html layout is top-down anyway.  :)

Reply:
The icons weren't being displayed on top of the banner if I didn't have the column in this. Maybe I could have achieved this a  different way.